src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -197,7 +198,8 @@ private string ReplaceUrlWithBase64Image(string html, string minifiedURL, bool m
                 base64 = $"data:application/x-font-{ext};charset=utf-8;base64,{base64}";
             }
 
-            html = html.Replace(minifiedURL, '"' + base64 + '"');
+            //In Libraryjs project, Webpack5 is removing the initial slash "/" when using resource files so for example in the html we have the string like "/resources/image.svg" then we need to remove the first slash (the first char) otherwise it won't be found and replaced by the base64 content.
+            html = html.Replace(minifiedURL.Replace(minfiedURLHtmlReplacement, minfiedURLHtmlReplacement.Remove(0, 1)), '"' + base64 + '"');
```

Comment:
I think https://learn.microsoft.com/en-us/dotnet/api/system.string.trimstart?view=net-7.0 (trim.start) might be a bit more idiomatic and clearer to someone seeing this for the first time as it will include the character you are trying to remove.

Reply:
Fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13417/commits/0ee6320b601fc8bf1976063b2db22dcf4855f6b2